conf/distro/rte.conf

```diff
@@ -108,3 +108,7 @@ require conf/distro/include/poky-world-exclude.inc
 require conf/distro/include/no-static-libs.inc
 require conf/distro/include/yocto-uninative.inc
 INHERIT += "uninative"
+
+# U-boot
+PREFERRED_VERSION_u-boot = "2020.04"
+BBMASK = "meta-sunxi/recipes-bsp/u-boot/u-boot_%.bbappend"
```

Comment:
@Cezarus27 why we set BBMASK here? I think we should use that bbappend from meta-sunxi to create boot.scr file and use patch to enable eMMC support.